crates/voicevox_core/src/synthesizer.rs

```diff
@@ -108,6 +108,22 @@ pub(crate) mod blocking {
 
     const DEFAULT_SAMPLING_RATE: u32 = 24000;
 
+    /// ユーザに渡す中間生成物。
+    pub struct Audio {
+        /// (フレーム数, 特徴数)の形を持つ音声特徴量。
+        pub internal_state: ndarray::Array2<f32>,
+        /// 生成時に指定したスタイル番号。
+        pub style_id: crate::StyleId,
+        /// workaround paddingを除いた音声特徴量のフレーム数。
+        pub length: usize,
+        /// サンプリングレート。全体の秒数は`length / sampling_rate`で表せる。
+        pub sampling_rate: f32,
+        /// workaroundとして付け足されているパディング長。
+        pub padding_length: usize,
+        /// 生成時に利用したクエリ。
+        pub audio_query: AudioQuery,
```

Comment:
ここら辺の可視性はPyO3とかと揃えた方がよいかなと。

```suggestion
        /// (フレーム数, 特徴数)の形を持つ音声特徴量。
        internal_state: ndarray::Array2<f32>,
        /// 生成時に指定したスタイル番号。
        style_id: crate::StyleId,
        /// workaround paddingを除いた音声特徴量のフレーム数。
        pub length: usize,
        /// サンプリングレート。全体の秒数は`length / sampling_rate`で表せる。
        pub sampling_rate: f32,
        /// workaroundとして付け足されているパディング長。
        padding_length: usize,
        /// 生成時に利用したクエリ。
        audio_query: AudioQuery,
```